Branch-SDK/src/io/branch/referral/BranchConfig.java

```diff
@@ -0,0 +1,8 @@
+package io.branch.referral;
+
+/*
+ *
+ */
+public class BranchConfig {
```

Comment:
It seems like this would be an appropriate place for things like the release number -- it's used for that in iOS. It also sets the base server url depending on whether you're in debug.